src/invision/components/header/header.js

```diff
@@ -1,16 +1,23 @@
 import React from 'react';
-
+import MoviePopup from "../add-movie/add-movie";
 import './header.scss';
 
 export default class Header extends React.Component {
+  constructor() {
+    super();
+    this.visible = false;
+  }
+  showAddMoviePopup() {
+    this.visible = true
```

Comment:
I am changing value of visible to true and passed visible property to MoviePopup component.

Reply:
You should use **state** there. Look at the following links:
https://reactjs.org/docs/state-and-lifecycle.html#using-state-correctly
https://www.w3schools.com/REACT/react_state.asp